packages/fxa-settings/README.md

```diff
@@ -14,7 +14,54 @@ You can import React components into this project. This is currently restricted
+
+```html
+<p class="text-lg text-red-600 background-white text-center p-10">
+  Hello, world!
+</p>
+```
+
+It's fairly straightforward once you understand how size, scales and other sequential patterns work. The [Tailwind docs](https://tailwindcss.com/docs) are very helpful in understanding these concepts (and they have an excellent search), and there are plugins for various IDEs ([VS Code](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss), [IntelliJ IDEs](https://plugins.jetbrains.com/plugin/12074-tailwindcss), [Vim](https://github.com/iamcco/coc-tailwindcss)).
+
+Here are a few additional things to keep in mind when developing with Tailwind:
+
+- We overwrite some of the default values provided by Tailwind, such as color and breakpoints. Refer to [tailwind.config.js](./tailwind.config.js) to see how values vary.
+- You do not need to import any custom or built-in styles/dependencies to use Tailwind. Just write your class names.
+- When it comes to time to create a production build we use PostCSS and PurgeCSS to strip out unused styles. PurgeCSS will look through all of the TSX files and identify which class names to keep styles for. In order for this to work properly it's important to avoid dynamic class names:
+
+  ```tsx
+  // Bad
+  const Button = ({ textColor, children }) => {
+    return <button className={`text-${textColor}`}>{children}</button>;
+  };
+  <Button textColor="green-500">Hello, world</Button>;
+
+  // Good!
+  const Button = ({ textColor, children }) => {
+    return <button className={textColor}>{children}</button>;
+  };
+  <Button textColor="text-green-500">Hello, world</Button>;
+  ```
+
+- If you find a class name is getting stripped out erroneously in production builds, it can be [whitelisted](https://purgecss.com/whitelisting.html) in [tailwind.config.js](./tailwind.config.js) or in the stylesheet itself by placing `/* purgecss ignore */` directly above the selector. Please use sparingly.
+
+#### Custom styles
+
+In the event you need to write styles that Tailwind is not able to affectively cover you should try to keep the following guidelines in mind:
+
+- SCSS is available.
+- PurgeCSS does not currently strip styles written outside of `src/styles/tailwind.css`, so it's up to you to keep styles limited to what you're using in your component.
+- Custom styles should be restricted to the component they are being used in. Typically, this is just an `index.scss` file adjacent to your component's `index.tsx` and then imported.
+- Because custom styles, once imported, are included in the global styles you should still be careful about how you're naming classes to avoid collisions. Generally you should match the class name to the name of your component. If you're using a shared component you could name according to its usage (e.g. using a shared UnitRow to display secondary emails could be `secondary-emails-row`).
```

Comment:
I would recommend this being `unit-row-secondary-emails`, [component]-[descriptor].